cluster/common.sh

```diff
@@ -699,6 +714,11 @@ EOF
 KUBEPROXY_TEST_LOG_LEVEL: $(yaml-quote ${KUBEPROXY_TEST_LOG_LEVEL})
 EOF
     fi
+    if [ -n "${KUBEPROXY_FEATURE_CONFIG:-}" ]; then
+      cat >>$file <<EOF
+KUBEPROXY_FEATURE_CONFIG: $(yaml-quote ${KUBEPROXY_FEATURE_CONFIG})
```

Comment:
I originally planned to do kubelet, but it was suggested in proposal that kubelet feature config should be handled by #29459. However it doesn't sound like that proposal will be implemented in the near term. @vishh wdyt of adding `--feature-config` to kubelet now and folding it into the dynamic config implementation when it's ready?